client/state/themes/actions/activate-theme.js

```diff
@@ -55,6 +59,23 @@ export function activateTheme( themeId, siteId, source = 'unknown', purchased =
 				dispatch(
 					themeActivated( themeStylesheet, siteId, source, purchased, styleVariationSlug )
 				);
+
+				if ( showSuccessNotice ) {
+					dispatch(
+						successNotice(
+							translate( 'The %(themeName)s theme is activated successfully!', {
+								args: { themeName: theme.name },
+							} ),
+							{
+								button: translate( 'View site' ),
+								href: getSiteUrl( getState(), siteId ),
+								duration: 10000,
+								showDismiss: false,
+							}
+						)
+					);
+				}
```

Comment:
Should we (can we) keep the notice since it includes the next step? I believe we should keep it visible so that users can follow the link. WDYT? (Sorry for my late review!) @arthur791004 @lsl 
p1725899556548679/1725851524.314329-slack-CRWCHQGUB

Reply:
IMO 10s or maybe 15s is enough to follow the action there if they want. Personally, it seems a bit annoying if it's always there and I have to dismiss it manually. We can try 10s first and then iterate it according to the feedback. Does it make sense?

Reply:
> We can try 10s first and then iterate it according to the feedback

Sounds good to me. Let's start with 10 seconds!